geolocation-API/watchPosition_TypeError.https.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>Geolocation Test: watchPosition TypeError tests</title>
+<link rel="help" href="http://www.w3.org/TR/geolocation-API/" />
```

Comment:
while here, it's probably a good idea to switch to https and s/geolocation-API/geolocation/

Reply:
ah yes... will do that globally.